### ISSUE TITLE
Missing global API version

### ISSUE THREAD

**Issue by lmazuel** [Azure/autorest.modelerfour#94]
https://github.com/Azure/autorest.testserver/blob/a5e6e2ef725df87c8981116836531a2f10be9128/swagger/azure-special-properties.json#L362-L371

![image](https://user-images.githubusercontent.com/1050156/70938891-f5d74680-1ffb-11ea-85d2-f53e6ea51d5e.png)

M4 4.2.99, Core 3.0.6179, CMD 3.0.6149

**Comment by fearthecowboy** [comment]
see #112

**Comment by fearthecowboy** [comment]
This should now be in place too. 

https://github.com/Azure/autorest.modelerfour/blob/f80448d1bbabcf764bdee2957c6d5ef012f9aa9e/modelerfour/test/scenarios/azure-special-properties/modeler.yaml#L948-L950

**Comment by lmazuel** [comment]
@fearthecowboy still don't see it :(
```
version: 3.0.6185
use-extension:
  "@autorest/modelerfour": "4.3.121"
```
autorest-beta 3.0.6149

**Comment by lmazuel** [comment]
Fixed in 4.9
